src/data/testData.js

```diff
@@ -0,0 +1,3151 @@
+export let deathsTestData = [
```

Comment:
what about moving this test data to some directory like `data/__tests__/testData.js` to make it clear it is for tests only?

Reply:
Good suggestion.  I didn't see an existing spot.